portal/server/utils/csv-utils.js

```diff
@@ -13,6 +13,15 @@ const columnIndexToExcelColumn = (index) => {
   return result;
 };
 
+const excelColumnToColumnIndex = (column) => {
+  let result = 0;
+  for (let i = 0; i < column.length; i += 1) {
+    result *= 26;
+    result += column.charCodeAt(i) - 64;
```

Comment:
Could we give some names to the `26` and `64` constants to explain what they are?

Reply:
Added some comments, haven't named as thought comments would do a better job not sure if that's a red flag.
26 = number of characters in alphabet
64 = A is the 65th Unicode character so making A = 1